### ISSUE TITLE
The hugo-lightslider-example does not work anymore

### ISSUE THREAD

**Issue by abirger** [pcdummy/hugo-lightslider-example#2]
I did exactly what README said: cloned the repo, and ran Hugo with a local server. However, it does not work for me neither in Firefox nor in Chrome. I noticed that Lightslider's and Lightgallery's last updates have been done after the hugo-lightslider-example's one. Has something critical changed there?

Comparison of my local "index.html" file with the `http://hugo-lightslider.pc-dummy.net/` page source confirmed that they are identical except several baseURL-related references.

**Comment by jochumdev** [comment]
@abirger thanks for reporting! Will have a look on that right now.

**Comment by jochumdev** [comment]
@abirger hmm seems to work as expected. I use the latest master of [hugo](https://github.com/spf13/hugo#build-and-install-the-binaries-from-source-advanced-install)

Can you upload your result somewhere so i can have a look on it?

**Comment by jochumdev** [comment]
@abirger Thanks for your /public, my index.html is the same as yours. :/ Now idk what to do.

**Comment by abirger** [comment]
@pcdummy , Wow, it was fast!

You see, I've tried it in a different way: I forked it, and then cloned it. The results were almost the same with one more thing happened after running Hugo (btw, I used Hugo v0.14 & 0.13 - no difference):
- it does not work, i.e. it gives me just a bulleted list of pictures and thumbnails (all of the same big size);
- in the `index.html` file all strings with `localhost:1313` got one more slash after that, e.g. instead of
  `<link rel="stylesheet" type="text/css" href="http://localhost:1313/css/lightGallery.css" />` 
  I got  `<link rel="stylesheet" type="text/css" href="http://localhost:1313//css/lightGallery.css" />`. 
  I manually fixed the URLs but you may still see the error in the `index_original.html` file in the `/public`.

**Comment by abirger** [comment]
If your `index.html` is the same, there must be a difference in my local browser config. Will look at it again...

**Comment by abirger** [comment]
@pcdummy , btw, after running Hugo should the scripts in the `/public/js` folder still be hard links to the`/vendor` folder content, or should they be replaced with the real jscripts? In my case, `jquery.lightSlider.min.js` and `lightGallery.min.js` are still linked to the `/vendor` folder but shouldn't the `public`'s content be ready for rendering and self-sufficient without dependency on other folder's content?..

**Comment by abirger** [comment]
@pcdummy , I got it! The problem is in the links to the `.js`, `.css`, image and font files. In Windows, the hardlinks and symlinks work differently from Linux. I have never been able to figure out which type of those links actually works in Windows... 

Anyway, the bottom line: as soon as I replaced all the hardlinks in the `/public/*` folders with the real files and fixed these dreadful `//`, it started working! Thank you.
